fix: app cmd creates a blockchain with a wrong name on v0.13

There is something wrong with parsing Github URLs. Username is used as an app name.

```
gitpod /workspace/starport/docs $ starport app github.com/fadeev/voter --sdk-version launchpad

⭐️ Successfully created a Cosmos app 'fadeev'.
👉 Get started with the following commands:

 % cd fadeev
 % starport serve

NOTE: add --verbose flag for verbose (detailed) output.
gitpod /workspace/starport/docs $ starport app github.com/fadeev/voter --sdk-version launchpad

repository already exists
gitpod /workspace/starport/docs $ starport app github.com/ilgooz/voter --sdk-version launchpad

⭐️ Successfully created a Cosmos app 'ilgooz'.
👉 Get started with the following commands:

 % cd ilgooz
 % starport serve

NOTE: add --verbose flag for verbose (detailed) output.
gitpod /workspace/starport/docs $ starport app github.com/username/voter
username/query.proto:4:1: warning: Import google/api/annotations.proto is unused.
username/query.proto:5:1: warning: Import cosmos/base/query/v1beta1/pagination.proto is unused.
username/query.proto:4:1: warning: Import google/api/annotations.proto is unused.
username/query.proto:5:1: warning: Import cosmos/base/query/v1beta1/pagination.proto is unused.

⭐️ Successfully created a Cosmos app 'username'.
👉 Get started with the following commands:

 % cd username
 % starport serve

NOTE: add --verbose flag for verbose (detailed) output.
```

--- a/go.mod
+++ b/go.mod
@@ -45,11 +45,11 @@ require (
 	github.com/stretchr/testify v1.6.1
 	github.com/tendermint/spn v0.0.0-20201215081711-b9ec9286ed83
 	github.com/tendermint/tendermint v0.34.0-rc6
-	golang.org/x/crypto v0.0.0-20201217014255-9d1352758620 // indirect
+	golang.org/x/crypto v0.0.0-20201221181555-eec23a3978ad // indirect
 	golang.org/x/mod v0.4.0
 	golang.org/x/net v0.0.0-20201021035429-f5854403a974 // indirect
 	golang.org/x/sync v0.0.0-20201207232520-09787c993a3a
-	golang.org/x/sys v0.0.0-20201221093633-bc327ba9c2f0 // indirect
+	golang.org/x/sys v0.0.0-20201223074533-0d417f636930 // indirect
 	golang.org/x/term v0.0.0-20201210144234-2321bbc49cbf // indirect
 	google.golang.org/genproto v0.0.0-20201019141844-1ed22bb0c154 // indirect
 	google.golang.org/grpc v1.33.0

--- a/starport/pkg/gomodulepath/gomodulepath.go
+++ b/starport/pkg/gomodulepath/gomodulepath.go
@@ -9,6 +9,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/tendermint/starport/starport/pkg/gomodule"
 	"golang.org/x/mod/module"
+	"golang.org/x/mod/semver"
 )
 
 // Path represents a Go module's path.
@@ -77,7 +78,7 @@ func validatePackageName(name string) error {
 func root(path string) string {
 	sp := strings.Split(path, "/")
 	name := sp[len(sp)-1]
-	if strings.HasPrefix(name, "v") { // omit versions.
+	if semver.IsValid(name) { // omit versions.
 		name = sp[len(sp)-2]
 	}
 	return name

